WordPress/gutenberg
# Block Search: Fixed the issue of width not updating to 25%

## What?
Fixes: #60414

## Why?
In the Search block, when we put width 25% in the editor, it doesn't get applied on the frontend.

## How?
I noticed that the search block has CSS set in the editor to ensure a minimum width of 220px because if the width goes beyond this value, it doesn't look good in terms of styling. Therefore, adding this to the frontend resolves the issue.

## Testing Instructions
1. Open/Edit a Post
2. Add Search Block
3. Apply width - 25%, 50%, 75%, 100% or in px and see the result in both frontend and editor.

### Testing Instructions for Keyboard
NIL
